src/main/java/org/springframework/cloud/deployer/spi/kubernetes/AbstractKubernetesDeployer.java

```diff
@@ -444,5 +451,16 @@ private String getDeploymentServiceAccountName(AppDeploymentRequest request) {
 		}
 
 		return deploymentServiceAccountName;
```

Comment:
Looks like a missing brackets after the return leading to compilation failure.

Reply:
rebase error.. fixed